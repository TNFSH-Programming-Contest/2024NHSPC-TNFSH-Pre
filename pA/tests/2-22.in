692944755333763227 780151533888542673 859379248576604032
692944755333763227 780151533888542673 859379248576604032
