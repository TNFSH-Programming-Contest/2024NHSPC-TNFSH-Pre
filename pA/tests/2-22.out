2332475537798909932
